classes/Product.php

```diff
@@ -7719,28 +7564,14 @@ public function updateWs($null_values = false)
      *
      * @param int $id_product Product identifier
      * @param int $id_product_attribute Attribute identifier
-     * @param int $id_warehouse Warehouse identifier
+     * @param int $id_warehouse Warehouse identifier - not used anymore
      * @param int|null $id_shop Shop identifier
      *
      * @return int real_quantity
      */
     public static function getRealQuantity($id_product, $id_product_attribute = 0, $id_warehouse = 0, $id_shop = null)
```

Comment:
shouldn't we deprecate this method( and a trigger a deprecation error)?

Reply:
Well, although not called in the core, the remaining part is a legit call, so I let it live. But we can maybe sometime do a pass and decide what unused methods to remove?

Reply:
Done :-)